packages/regl-worldview/src/commands/GLText.js

```diff
@@ -416,9 +449,21 @@ function makeTextCommand() {
 }
 
 export default function GLText(props: Props) {
+  const context = useContext(WorldviewReactContext);
+  const { dimension } = context;
+
   const [command] = useState(() => makeTextCommand());
   // HACK: Worldview doesn't provide an easy way to pass a command-level prop into the regl commands,
   // so just attach it to the command object for now.
   command.autoBackgroundColor = props.autoBackgroundColor;
+
+  command.scaleInvariant = !!props.scaleInvariantFontSize;
```

Comment:
nit: what about `scaleInvariantFontSize={0}`? (Not that this would be desirable anyway)

Reply:
A value of 0 (zero) will disable scale invariance and a negative one will flip the Y coordinate. Like you said, I'm not sure if that's a valid use case, but the size effects are visible. Should I constraint the sizes to positive values only?

Reply:
Or just use
```suggestion
  command.scaleInvariant = props.scaleInvariantFontSize != null;
```
here?

Reply:
@jtbandes I made the change you suggested. Now it handles the scaleInvariantFontSize properly.